it/mail/chrome/messenger/am-mdn.dtd

```diff
@@ -1,24 +1,24 @@
 <!ENTITY pane.title                   "Ricevute di ritorno">
-<!ENTITY useGlobalPrefs.label         "Per questo account utilizza le preferenze globali">
+<!ENTITY useGlobalPrefs.label         "Utilizza le mie preferenze globali per le ricevute di ritorno per questo account">
 <!ENTITY useGlobalPrefs.accesskey     "u">
-<!ENTITY useCustomPrefs.label         "Personalizza le opzioni per le ricevute di ritorno di questo account">
+<!ENTITY useCustomPrefs.label         "Personalizza le ricevute di ritorno per questo account">
 <!ENTITY useCustomPrefs.accesskey     "P">
-<!ENTITY requestReceipt.label         "Quando si spediscono i messaggi chiedi sempre la Ricevuta di ritorno">
+<!ENTITY requestReceipt.label         "Durante l'invio dei messaggi, richiedi sempre una ricevuta di ritorno">
 <!ENTITY requestReceipt.accesskey     "Q">
 <!ENTITY receiptArrive.label          "Quando arriva una ricevuta:">
-<!ENTITY leaveIt.label                "Lasciala nella posta in arrivo">
+<!ENTITY leaveIt.label                "Lasciala nella mia posta in arrivo">
```

Comment:
CORRECTIONS NEEDED. 

The possessive pronoun "mia" is redundant. It's obviously _mine_ inbox, and not someone else's... that's why I would remove "mia".